packages/solid/package.json

```diff
@@ -12,29 +12,45 @@
   "files": [
     "cjs",
     "esm",
-    "types",
+    "index.d.ts",
+    "index.d.ts.map",
+    "solidrouter.d.ts",
+    "solidrouter.d.ts.map",
     "types-ts3.8"
   ],
   "main": "build/cjs/index.js",
   "module": "build/esm/index.js",
-  "types": "build/types/index.d.ts",
+  "types": "build/index.d.ts",
   "exports": {
     "./package.json": "./package.json",
     ".": {
       "import": {
-        "types": "./build/types/index.d.ts",
-        "default": "./build/esm/index.js"
+        "default": "./build/esm/index.js",
```

Comment:
This has to be the other way around, see https://github.com/getsentry/sentry-javascript/pull/12404 - is there a reason you reordered this? types should always come first!

Reply:
Updated, thanks!